### DISCUSSION TITLE
connect to the lima vm from host by IP or domain

### DISCUSSION THREAD

**Discussion by ysdlb** [lima-vm/lima#1109] [Q&A] [answered]
I can talk to the host from lima vm by `host.lima.internal` or  `host.docker.internal`.
Now i want connect to the lima vm from host, and i don't want use the shared port number. 
I don't know how to do that.

**Comment by AkihiroSuda** [comment, marked as answer]
The following configuration assigns "real" IP addresses to the VM:
```console
$ limactl sudoers | sudo tee /etc/sudoers.d/lima

$ limactl start --name=default template://vmnet
```
Dependency: https://github.com/lima-vm/socket_vmnet

See also https://github.com/lima-vm/lima/blob/master/docs/network.md
